presto-docs/src/main/sphinx/connector/base-arrow-flight.rst

```diff
@@ -3,7 +3,6 @@
 Arrow Flight Connector
 ======================
 This connector allows querying multiple data sources that are supported by an Arrow Flight server.
-Apache Arrow enhances performance and efficiency in data-intensive applications through its columnar memory layout, zero-copy reads, vectorized execution, cross-language interoperability, rich data type support, and optimization for modern hardware. These features collectively reduce overhead, improve data processing speeds, and facilitate seamless data exchange between different systems and languages.
```

Comment:
Can you a give a brief overview and link to the official documentation.

Reply:
done